templates/macros/extended_footer.html

```diff
@@ -2,41 +2,35 @@
    <div class="extended_footer">
       {%- if config.generate_feed and config.extra.show_rss_badge -%}
       <a href="{{ get_url(path=config.feed_filename) }}" style="text-decoration:none">
-         {%- set rss_image =  resize_image(path="/assets/social/rss.png", width=22, height=22, op="scale", format="png") -%}
+         {%- set rss_image =  resize_image(path="../static/assets/social/rss.png", width=22, height=22, op="scale", format="png") -%}
          <img src="{{ rss_image.url }}" style="display:inline" alt="RSS Badge" />
       </a>
       {%- endif -%}
 
       {%- if config.extra.github_profile -%}
       <a href="{{ config.extra.github_profile}}" style="text-decoration:none">
-         {%- set gh_image =  resize_image(path="/assets/social/github.png", width=22, height=22, op="scale", format="png") -%}
+         {%- set gh_image =  resize_image(path="../static/assets/social/github.png", width=22, height=22, op="scale", format="png") -%}
          <img src="{{ gh_image.url }}" style="display:inline" alt="Github Profile Badge" />
```

Comment:
We would have to change `gh_image.url` to `gh_image` to have it work with Zola v0.13 (used by CI), but that wouldn't work with Zola v0.14 and onwards.
It seems moving CI to a new version is the best option? Since the entire theme does build with Zola v0.15.2 already.